app/models/protocol_authorizer.rb

```diff
@@ -28,8 +28,7 @@ def can_edit?
     # NOTE @can_edit memoized; use #nil? since @can_edit is a boolean.
     # !! maps truthy and falsey values to true and false
     if @can_edit.nil?
-      @can_edit = !!(@protocol && @identity &&
-        (roles_for_edit.any? || administrated_ssrs.any?))
+      @can_edit = !!(@protocol && @identity && roles_for_edit.any?)
```

Comment:
This admin logic is wrong, and besides, admins can't edit unless they have a role for edit